corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html

```diff
@@ -70,6 +70,10 @@ <h4 class="modal-title">
                     <div class="col-xs-12 col-sm-9">
                       <select class="form-control" data-bind="value: appearance">
                         <option value="">{% trans "Text" %}</option>
+                        {% if request|toggle_enabled:'USH_CASE_CLAIM_UPDATES' %}
+                          {# flag this one because it only works on web apps #}
```

Comment:
Forgive my case search ignorance, but I'm having trouble interpreting this. Is it that setting a specific date for this case search config only works in web apps? And to ensure it is only available to projects with web apps enabled, we rely on this `USH_CASE_CLAIM_UPDATES` flag? Is there not a better way to make this available to any project that has web apps enabled?

Reply:
Yes, setting a specific date only works on web apps, since I haven't updated the android UI code. We don't have a good automated way to distinguish between mobile projects and web apps projects, this flag will just limit it to USH projects. It's possible I should just remove the comment.

Reply:
Ah right! Makes sense. I like having the comment there, but might suggest:
```suggestion
                        {% if request|toggle_enabled:'USH_CASE_CLAIM_UPDATES' %}
                          {# limit to USH since datepicker is only available on web apps #}
```
Definitely a nit though, so feel free to ignore.